src/os_handlers.eliom

```diff
@@ -295,7 +295,7 @@ let action_link_handler_common akey =
       else Lwt.return_unit
     in
     let%lwt () =
-      if action = `AccountActivation
+      if action = `AccountActivation || action = `PasswordReset
```

Comment:
(preventing registering an email that's already been through the registration process) => (validate emails for which we require a password reset)